src/cli/utils/sharepoint.go

```diff
@@ -66,6 +66,33 @@ func MakeSharePointOpts(cmd *cobra.Command) SharePointOpts {
 	}
 }
 
+func SharePointAllowedCategories() map[string]struct{} {
```

Comment:
Correct me if I'm wrong, because I can't remember if we made the decision (during a sync meeting) for `Lists` or `Conversations`, but did we want to actually expose lists here, or have it be hidden/not part of the backup unless specified?  I thought we picked the latter while we're finishing up development?

Reply:
I believe the intention was to merge basic backup and restore functionality for lists to main and then work on other supports like: Delta, Metadata,

That is the reason we need to hide lists untill atleast `Incrementals` are in place.

Sure I can comment it here for time being, but If we were to test from cli, we can uncomment it that time. Does that work?